doc/infrastructures.rst

```diff
@@ -50,17 +50,51 @@ It is strongly recommended to associate labels to your hosts. Labels allow to fi
 
 .. _yorc_infras_hostspool_filters_section:
 
-Filters Grammar
-^^^^^^^^^^^^^^^
+Filter on label existence
```

Comment:
Maybe some doc about composite filters is missing ?

Reply:
Composite filters are not exposed to user currently, it would have been too costly to implement them into the grammar